### DISCUSSION TITLE
Standup (Friday, May 13, 2022)

### DISCUSSION THREAD

**Discussion by CarlosCRG19** [CarlosCRG19/ijalti-peers#58] [General]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:
```
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -

**Shoutouts**:
 - @username for x
 ```

**Comment by CristianCazares** [comment]
**What did you achieve in the last 24 hours?**:
 - Other stuff

**What are your priorities for the next 24 hours?**:
 - Other stuff, sadly

**Blockers**:
 - Semana Tec
 - Optativa stuff

**Comment by CarlosCRG19** [comment]
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - Deploy of the app

**Blockers**:
 - Worked on environment variables, documented about heroku

**Shoutouts**:
 - @DavBot02 for checking the pending pr

**Comment by DavBot02** [comment]
**What did you achieve in the last 24 hours?**:
 - Input for salary now accepts only digits

**What are your priorities for the next 24 hours?**:
 - Not sure

**Blockers**:
 - Last episode of KS

**Shoutouts**:
 - @everyone ily
